node/subsystem-util/src/lib.rs

```diff
@@ -82,35 +85,38 @@ pub const JOB_GRACEFUL_STOP_DURATION: Duration = Duration::from_secs(1);
 /// Capacity of channels to and from individual jobs
 pub const JOB_CHANNEL_CAPACITY: usize = 64;
 
-
 /// Utility errors
-#[derive(Debug, derive_more::From)]
+#[derive(Debug, Error)]
 pub enum Error {
 	/// Attempted to send or receive on a oneshot channel which had been canceled
-	#[from]
-	Oneshot(oneshot::Canceled),
+	#[error(transparent)]
+	Oneshot(#[from] oneshot::Canceled),
 	/// Attempted to send on a MPSC channel which has been canceled
-	#[from]
-	Mpsc(mpsc::SendError),
+	#[error(transparent)]
+	Mpsc(#[from] mpsc::SendError),
 	/// A subsystem error
-	#[from]
-	Subsystem(SubsystemError),
+	#[error(transparent)]
+	Subsystem(#[from] SubsystemError),
 	/// An error in the Chain API.
-	#[from]
-	ChainApi(ChainApiError),
+	#[error(transparent)]
+	ChainApi(#[from] ChainApiError),
 	/// An error in the Runtime API.
-	#[from]
-	RuntimeApi(RuntimeApiError),
+	#[error(transparent)]
+	RuntimeApi(#[from] RuntimeApiError),
 	/// The type system wants this even though it doesn't make sense
-	#[from]
-	Infallible(std::convert::Infallible),
+	#[error(transparent)]
+	Infallible(#[from] std::convert::Infallible),
 	/// Attempted to convert from an AllMessages to a FromJob, and failed.
+	#[error("AllMessage not relevant to Job")]
 	SenderConversion(String),
 	/// The local node is not a validator.
+	#[error("Node is not a validator")]
 	NotAValidator,
 	/// The desired job is not present in the jobs list.
+	#[error("Relay parent {0} not of interest")]
 	JobNotFound(Hash),
 	/// Already forwarding errors to another sender
+	#[error("AlreadyForwarding")]
```

Comment:
Should we expand this?

Reply:
@coriolinus I would value your input!

Reply:
I don't think that's necessary. If this error is ever thrown, it's a logical error in your code, and you can debug by simply searching your diff for additions or loops of `forwards_errors`.